src/main/java/org/candlepin/subscriptions/tally/TallySnapshotController.java

```diff
@@ -155,4 +160,13 @@ private void attemptCloudigradeEnrichment(List<String> accounts,
         }
     }
 
+    private static boolean isCombiningRollupStrategySupported(
+        Map.Entry<OffsetDateTime, AccountUsageCalculation> usageCalculations) {
+
+        var calculatedProducts = usageCalculations.getValue().getProducts();
+
+        return calculatedProducts.contains(ProductId.OPENSHIFT_METRICS.toString()) ||
```

Comment:
Is this something we should just have in the product profile registry?  We could just add it as a new attribute (true by default) and update the `Openshift-metrics` profile.

Reply:
I'm thinking yes.  I've actually got a whole bunch of ideas floating around in my brain about what we could/should add to the product profile registry.  I'm in mode "get this to work without bugs" right now and want to talk about the product profile registry in one of those dev backend syncs soon™

Reply:
:+1: